web/src/test/java/org/cbioportal/web/api/ApiControllerTest.java

```diff
@@ -543,7 +544,7 @@ public void geneticProfileDataTest4() throws Exception {
         GeneticProfile geneticProfile_brca_tcga_mutations = new GeneticProfile();
```

Comment:
Definitely yes. I created this ugliness (first time using mockito) ... and there must be a better way of specifying your mock objects than hand constructing them like this.